app/src/test/java/de/westnordost/streetcomplete/quests/opening_hours/AddOpeningHoursTest.kt

```diff
@@ -88,6 +132,40 @@ class AddOpeningHoursTest {
         )
     }
 
+    @Test fun `apply always open answer when it was not signed before`() {
+        questType.verifyAnswer(
+            mapOf("opening_hours:signed" to "no"),
+            AlwaysOpen,
+            StringMapEntryAdd("opening_hours", "24/7"),
+            StringMapEntryDelete("opening_hours:signed", "no")
+        )
+    }
+
+    @Test fun `apply always open answer when it was not signed but present before`() {
+        questType.verifyAnswer(
+            mapOf(
+                "opening_hours" to "24/7",
+                "opening_hours:signed" to "no"
+            ),
+            AlwaysOpen,
+            StringMapEntryModify("opening_hours", "24/7", "24/7"),
+            StringMapEntryAdd("check_date:opening_hours", LocalDate.now().toCheckDateString()),
```

Comment:
Plus here, again we've only checked the unsigned aspect.

Reply:
What? No. here we confirmed that it is open 24/7

Reply:
Oops yeah.